syntax = "proto3";
package Protocol;


import "Enum.proto";
import "Struct.proto";


// path = Login
message C_Login
{
}

// path = Login
message S_Login
{
	bool success = 1;
	repeated Player players = 2;
}

// path = Chat
message C_Chat
{
	string msg = 1;
}

// path = Chat
message S_Chat
{
	uint64 playerId = 1;
	string msg = 2;
}